Peaky/TestOutcome.cs

```diff
@@ -0,0 +1,11 @@
+namespace Peaky
+{
+    internal enum TestOutcome
```

Comment:
In TRX there's also a `NOTEXECUTED` value, which is what gets written for example when a test is marked as "skip".